app/javascript/packages/document-capture/context/analytics.tsx

```diff
@@ -67,18 +66,10 @@ export function AnalyticsContextProvider({ children, trackEvent }: AnalyticsCont
 
     setSubmitEventMetadataState(DEFAULT_EVENT_METADATA);
   };
-  const { inPersonCtaVariantActive } = useContext(InPersonContext);
 
-  const extraAnalyticsAttributes = (stepName) => {
-    const extra: EventMetadata = { ...DEFAULT_EVENT_METADATA };
-    if (stepName === 'prepare' || stepName === 'location') {
-      extra.in_person_cta_variant = inPersonCtaVariantActive;
-    }
-    return extra;
-  };
   const trackVisitEvent: TrackVisitEvent = (stepName) => {
     if (LOGGED_STEPS.includes(stepName)) {
-      trackEvent(`IdV: ${stepName} visited`, extraAnalyticsAttributes(stepName));
+      trackEvent(`IdV: ${stepName} visited`, {});
```

Comment:
Second argument is optional if we don't have anything to pass anymore.

```suggestion
      trackEvent(`IdV: ${stepName} visited`);
```